src/systemctl/systemctl.c

```diff
@@ -4253,7 +4257,24 @@ static void print_status_info(
         else
                 printf("\n");
 
-        if (endswith(i->id, ".timer")) {
+        is_timer = endswith(i->id, ".timer");
+
+        if (!is_timer && !strv_isempty(i->triggered_by)) {
+                char **trigger;
+                bool found = false;
+
+                STRV_FOREACH(trigger, i->triggered_by) {
+                        if (endswith(*trigger, ".timer")) {
+                                if (!found) {
+                                        printf("Activator: Timers\n");
```

Comment:
We say "Trigger" in .timer unit. For symmetry, I think we should say "TriggeredBy" here. We don't use the term "activator" anywhere else.

If the triggering unit is not active, it will still be shown here. We need to indicate this somehow, maybe even by showing a color dot to indicate the unit status. (There is an assymetry here: when we are showing a .timer unit, and we say 'Triggers: foo.service', it doesn't matter that much whether foo.service is active or not. It will be started by trigger when the time comes either way. But when we are showing the .service unit, if foo.trigger is inactive, no triggering will happen.)